lib/constants.js

```diff
@@ -0,0 +1,13 @@
+export const BUTTER_CLIENT_VERSION = '2.0.0'; // {x-release-please-version}
```

Comment:
I'm not familliar with `x-release-please-version`. Do we need to use it instead of `'2.0.0'`?

Reply:
neither am I but I think its an automated thing during publishing, as Jake mentioned in the weekly meeting today